# Optimisation and Validation of a Metagenomic-Based Sequencing Protocol for HIV Viral Load Quantification Using Mengovirus as an Internal Control.

This project evaluates the efficacy of a novel metagenomic sequencing protocol for HIV detection and quantification. By integrating qPCR data with next-generation sequencing (NGS) metrics, this work aims to improve the accuracy of viral load estimations from clinical samples.

## Project Overview

### Key Objectives
- **Primer Design and Optimization**: Designed and optimized primers for HIV and an internal control virus (Mengovirus) to ensure consistent amplification across various samples.
- **qPCR Validation**: Conducted qPCR experiments to validate the primers and assess amplification efficiency and specificity.
- **Bioinformatics Analysis**: Processed and interpreted raw sequencing data, exploring correlations between Ct values and sequencing coverage.

### Methodology
- Utilised SYBR Green-based RT-qPCR assays to assess the specificity of primers across HIV-free and HIV-positive samples.
- Performed bioinformatics analyses using Python to analyze qPCR data alongside sequencing metrics.
- Developed and validated a multiple linear regression model to refine sequencing-derived viral load estimates.

## Results and Implications
- Demonstrated the potential of integrating qPCR with NGS for improved viral load estimation.
- Provided a scalable approach adaptable to other pathogen detection protocols.

## Collaborators and Acknowledgements
This work was conducted under the supervision of Dr. David Bonsall, with guidance from George McIntyre-Cockett and Dr. Nicholas Grayson. I am grateful to the Bonsall group, Dr. Sandra E. Chaudron, Dr. Chris Wymant, and the MSc Genomic Medicine course leads for their support and contributions.
